### PR TITLE
W-14038258-TracingOverview-SY

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/traces-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//insert link to data exporter eventually
+
+The Traces feature is available only for organizations that have a Titanium subscription or the Advanced pricing package for Anypoint Platform.
+
+== Before You Begin
+
+Before you can use traces, ensure that you have:
+
+* A Titanium subscription or the Advanced pricing package for Anypoint Platform
+* Mule Runtime version 4.5 or later
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
* Mule Runtime version 4.6 or later
```

File: modules/ROOT/pages/trace-details.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+= Trace Details
+
+Each trace available in the *Traces* overview page has metrics associated with it. Clicking a trace opens a *Trace Details* page and shows the origin of the initial request, associated entities, events associated with the request, and more. You can also see flows, logs, and events associated with the trace.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/trace-details.adoc]:
```suggestion
Each trace available in the *Traces* overview page has metrics associated with it. Clicking a trace opens a *Trace Details* page and shows the origin of the initial request, associated entities, events associated with the request, and more. You can also see flows, logs, and events associated with the trace in the form of _spans_. Each span contains details that explain the operations in the specific *Trace Detail* page.
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The Traces feature in Anypoint Monitoring enables you to observe requests for apps and APIs as they travel across services. _Traces_ describe the journey that the requests make throughout their pipelines of services. Each trace originates from an _entity_, such as an app or API, that is available in Anypoint Platform. The *Traces* overview page enables you to see a list of traces that are currently available in Anypoint Platform. You can use this data to troubleshoot requests that have errors or slow response times, optimize your resource allocation, and track your entity requests end-to-end. For example, an Ops administrator could use traces to search for a specific transaction and have a visualization for that transaction that includes only the services through which it progresses. The visibility enables transparent asset performance observation and management. To simplify these efforts, you can also export trace data to Open Telemetry or Splunk using the *Telemetry Exporter* feature. 
+
+//insert link to data exporter eventually
+
+The Traces feature is available only for organizations that have a Titanium subscription or the Advanced pricing package for Anypoint Platform.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
The Traces feature is available only for organizations that have the Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform.
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+//Sara - I would consider making this access topic a link in all the subsequent topics. I don't want to mess up anything you have going on here, so I won't change it but I think it would be nice!
+== Access Traces
+
+include::partial$include-nav-steps-no-permissions.adoc[]
+
+include::partial$include-nav-distributed-tracing.adoc[]
+
+== Filter Traces
+
+You can filter traces shown in the *Traces* overview page using a time period selector and queries.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
== Search for Traces

You can search for traces shown in the *Traces* overview page using a time period selector and queries. 
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+== Filter Traces
+
+You can filter traces shown in the *Traces* overview page using a time period selector and queries. 
+
+=== Filter Traces Using the Time Period Selector
+
+Use the time period selector to display traces for the last given period of time (such as the last 5 or 30 minutes) or over a given date and time range. Use the drop-down in the calendar icon to select the time period to display in a graph. Your setting applies across all traces in the *Traces* overview page, including the *Total Traces* chart. 
+
+Anypoint Monitoring retains up 30 days of trace data.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
=== Search for Traces Using the Time Period Selector

Use the time period selector to display traces for the last given period of time (such as the last 5 or 30 minutes) or over a given date and time range. Use the drop-down in the calendar icon to select the time period to display in a graph. Your setting applies across all traces in the *Traces* overview page, including the *Trace Volume* chart. 

Anypoint Monitoring retains up 30 days of trace data.
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+=== Filter Traces Using Queries
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
=== Search for Traces Using Queries
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+Custom Attribute:: Search transactions for a specific customer-defined attribute (for example, *Order ID*)
+Span Name:: Search for a specific span
+
+// needs editing
+
+If no traces appear after applying a query, limit your conditions. 
+
+== Trace Overview Metrics
+
+The *Traces* overview page shows the *Traces List* chart by default, which displays both the total number of traces in Anypoint Platform and the total number of traces that have errors associated with them. You can use the time period selector to select the span of time covered by the *Traces List* chart. From the *Traces List* chart, you can click the *Trace ID* of a specific trace to view more information the *Trace Details* page.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
The *Traces* overview page shows the *Trace Volume* chart by default, which displays both the total number of traces in Anypoint Platform and the total number of traces that have errors associated with them. You can use the time period selector to select the span of time covered by the *Trace Volume* chart. From the *Trace Volume* chart, you can click the *Trace ID* of a specific trace to view more information on the xref:trace-details.adoc[*Trace Details*] page. 
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+Span response time range:: filter and list all spans with response time range E.g. response time  > 30 ms or between 10 ms and 30 ms  
+Transaction start time:: Search all transactions for a specific time range
+Service name:: Filter all the transactions and spans for a specific service
+API Name:: Filter all the transactions/spans for a specific API
+Tags:: If a tag has been defined for an entity, it can be used to quickly search for transactions specific to the tag. 
+Flow name:: Search span details for a specific flow
+Custom Attribute:: Search transactions for a specific customer-defined attribute (for example, *Order ID*)
+Span Name:: Search for a specific span
+
+// needs editing
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
Use the search bar to select fields to query entities and their traces. You can query using the following criteria:

* `business_group_id`: The ID of the business group that contains entities for which you want to see traces. This query appears in the search bar by default.
* `environment_id`: The ID of the environment that contains entities for which you want to see traces. This query appears in the search bar by default.
* `business_group_name`: The name of the business group that contains entities for which you want to see traces. 
* `entry_api_version`: The version number of the API for which you want to see traces.
* `entry_entity_id`: The ID of the entity for which you want to see traces.
* `entry_entity_name`: The name of the entity for which you want to see traces.
* `entry_entity_type`: The type of entity for which you want to see traces. You can query apps or APIs.
* `entry_span_id`: The ID of a span contained in traces.
* `entry_span_name`: The name of a span contained in traces.
* `environment_name`: The name of the environment that contains entities for which you want to see traces.
* `http_status_code`: The HTTP status code associated with a trace. You can query 0, 200, or 500 codes.
* `trace_id`: The ID associated with a trace.

```

File: modules/ROOT/pages/traces-overview.adoc [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+//insert link to data exporter eventually
+
+The Traces feature is available only for organizations that have the Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform.
+
+== Before You Begin
+
+Before you can use traces, ensure that you have:
+
+* A Titanium subscription or the Advanced pricing package for Anypoint Platform
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
* The Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform // replace with partial
```

File: modules/ROOT/pages/trace-details.adoc [hunk 10]
```diff
@@ -0,0 +1,58 @@
+= Trace Details
+
+Each trace available in the *Traces* overview page has metrics associated with it. Clicking a trace opens a *Trace Details* page and shows the origin of the initial request, associated entities, events associated with the request, and more. You can also see flows, logs, and events associated with the trace in the form of _spans_. Each span contains details that explain the operations in the specific *Trace Detail* page.
```
Comment (emileemooreVT) [inline modules/ROOT/pages/trace-details.adoc]:
```suggestion
Each trace available in the *Traces* overview page contains associated metrics for you to examine. Clicking a trace opens a *Trace Details* page and shows the origin of the initial request, associated entities, events associated with the request, and more. Additionally, you can see flows, logs, and events associated with the trace in the form of _spans_. Each span contains details that explain the operations in the specific *Trace Detail* page.
```

File: modules/ROOT/pages/trace-details.adoc [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+include::partial$include-nav-distributed-tracing.adoc[]
+
+. If needed, use the search function to query the list of traces that appears in the *Traces* overview page.
+. Click the trace ID of the trace you want to examine.
+
+== Understand Trace Details
+
+Each trace detail page contains response times for spans at a glance. They also show the following contextual data:
+
+* *Status Code*: HTTP response status of the trace:
```
Comment (emileemooreVT) [inline modules/ROOT/pages/trace-details.adoc]: Should this have "The" first to be parallel or is this some weirdness when it applies to HTTP?

Comment (sarathecoubian) [reply modules/ROOT/pages/trace-details.adoc]:
```suggestion
* *Status Code*: The HTTP response status of the trace: 
```

File: modules/ROOT/pages/traces-overview.adoc [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+//insert link to data exporter eventually
+
+The Traces feature is available only for organizations that have the Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform.
+
+== Before You Begin
+
+Before you can use traces, ensure that you have:
+
+* The Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform // replace with partial
```
Comment (sarathecoubian) [inline modules/ROOT/pages/traces-overview.adoc]:
```suggestion
* The Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform
```

**Pull request merged**